src/lib.rs

```diff
@@ -46,6 +46,7 @@
 extern crate bincode;
 extern crate byteorder;
 extern crate crc;
+#[macro_use]
```

Comment:
lets remove this and actually import 😉 

We should also bump the min version to build this to 1.30

Reply:
There is already an issue about it so I'll leave that concern there.